geode-core/src/main/java/org/apache/geode/management/internal/MBeanJMXAdapter.java

```diff
@@ -72,16 +72,13 @@
   private DistributedMember distMember;
 
   /**
-   * log writer, or null if there is no distributed system available
-   */
-  // private LogWriterI18n logger = InternalDistributedSystem.getLoggerI18n();
-
-  /**
    * public constructor
    */
 
+  public static final int VALUE_NOT_AVAILABLE = -1;
```

Comment:
Should this live in VMStatsMonitor since that's the only place it appears to be used?

Reply:
The management pkg depends on the statistics pkg, but statistics should not have circular dependency on management. Moving the constant into VMStatsMonitor should resolve this.